pages/index.js

```diff
@@ -1,5 +1,17 @@
 import PlantList from "@/components/PlantList";
+import Footer from "@/components/Footer";
+import styled from "styled-components";
 
-export default function Homepage() {
-  return <PlantList />;
+export default function Homepage({ plants, toggleFavourite }) {
+  return (
+    <main>
```

Comment:
you have the `<main>` tag already in your Layout Component so please remove it here.